fe1-web/src/features/evoting/network/messages/__tests__/SetupElection.test.ts

```diff
@@ -120,14 +129,31 @@ describe('SetupElection', () => {
   it('fromJson should throw an error if the Json has incorrect action', () => {
     const obj = {
       object: ObjectType.ELECTION,
+      action: ActionType.NOTIFY_ADD,
+      id: electionId.toString(),
+      lao: mockLaoIdHash.toString(),
+      name: mockLaoName,
+      version: VERSION,
+      created_at: parseInt(TIMESTAMP.toString(), 10),
+      start_time: parseInt(TIMESTAMP.toString(), 10),
+      end_time: parseInt(CLOSE_TIMESTAMP.toString(), 10),
```

Comment:
These can be simply replaced by 
```suggestion
      created_at: TIMESTAMP.valueOf(),
      start_time: TIMESTAMP.valueOf(),
      end_time: CLOSE_TIMESTAMP.valueOf(),
```
But it's not a big issue.

Reply:
Thank you for the hint! I just pushed another commit to this PR regarding this 😊